pkg/bootstrap/docker/up.go

```diff
@@ -131,6 +132,7 @@ func NewCmdUp(name, fullName string, f *osclientcmd.Factory, out, errout io.Writ
 	cmd.Flags().StringVar(&config.DockerMachine, "docker-machine", "", "Specify the Docker machine to use")
 	cmd.Flags().StringVar(&config.ImageVersion, "version", "", "Specify the tag for OpenShift images")
 	cmd.Flags().StringVar(&config.Image, "image", "openshift/origin", "Specify the images to use for OpenShift")
+	cmd.Flags().StringVar(&config.ImageStreams, "image-streams", "origin centos7 image streams:examples/image-streams/image-streams-centos7.json", "Specify the image streams to use")
```

Comment:
having the example/default have spaces in it feels confusing (means users must quote the value if they type it themselves).  assuming we need a name at all, can it just be origin-centos7 or something?